packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/EditView/utils/schema.js

```diff
@@ -5,9 +5,13 @@ const schema = yup.object().shape({
   name: yup.string(translatedErrors.string).required(translatedErrors.required),
   type: yup
     .string(translatedErrors.string)
-    .oneOf(['read-only', 'full-access'])
+    .oneOf(['read-only', 'full-access', 'custom'])
```

Comment:
Just asking since I'm actually not sure how the front end works with the other modules and maybe this is impossible, but can these values be used directly from the constants at /core/admin/server/services/constants.js ?

Reply:
I don't think those files are available